python/cudf/cudf/core/column/decimal.py

```diff
@@ -80,19 +79,11 @@ def as_decimal_column(
     ) -> "cudf.core.column.DecimalColumn":
         if dtype == self.dtype:
             return self
-        result = libcudf.unary.cast(self, dtype)
-        if isinstance(dtype, cudf.core.dtypes.Decimal64Dtype):
-            result.dtype.precision = dtype.precision
-        return result
+        return libcudf.unary.cast(self, dtype)
```

Comment:
Just a quick side note that this is needed in https://github.com/rapidsai/cudf/pull/7716